### PR TITLE
Fix gender udf

### PR DESCRIPTION
**Pull request by ingkebil**
I added a default for the Gender UDF: unknown.

### CODE REVIEW

**Comment by ingkebil** [comment]
Ok to merge?

File: cglims/cli/commands.py [hunk 1]
```diff
@@ -194,17 +194,20 @@ def set_defaults(lims_sample):
     lims_sample.udf['Index number'] = 'NA'
     lims_sample.udf['Sample Buffer'] = 'NA'
     lims_sample.udf['Reference Genome Microbial'] = 'NA'
-    if lims_sample.udf['Gender']:
-        lims_sample.udf['Gender'] = lims_sample.udf['Gender'].upper()
-    if lims_sample.udf['Status']:
-        lims_sample.udf['Status'] = lims_sample.udf['Status'].lower()
+    gender = lims_sample.udf.get('Gender')
+    if gender:
+        lims_sample.udf['Gender'] = gender
```
Comment (robinandeer) [inline cglims/cli/commands.py]: aren't you just replacing the value by itself here? hehe
missing the `.lower()` call?

Comment (ingkebil) [inline cglims/cli/commands.py]: Yeah, I forgot the `upper()` call ;)

**Comment by ingkebil** [comment]
OK, addressed the request for changes.

I'll merge!

**Pull request merged**